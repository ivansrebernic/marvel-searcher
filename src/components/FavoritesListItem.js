import React from 'react'
import styled from 'styled-components'

const FavoriteItemStyled = styled.li`
    z-index:1000;   
    font-size:24px;
    margin:0.5rem;
    display:flex;
    justify-content:space-between;
    align-items:center;
    height:2rem;


    &:hover{
        cursor:pointer;
        background-color:#E5E5E5;
    }
`
function FavoriteItem(props) {

    return (
        <FavoriteItemStyled onClick={props.returnQuery}>
            {props.children}
        </FavoriteItemStyled>
    )

}

export default FavoriteItem