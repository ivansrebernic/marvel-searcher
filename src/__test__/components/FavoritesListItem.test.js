import React from 'react'
import { shallow } from 'enzyme'
import FavoritesListItem from '../../components/FavoritesListItem'


describe('<FavoritesListItem>', () => {
    it('Renders correctly', () => {
        const wrapper = shallow(<FavoritesListItem></FavoritesListItem>)
        expect(wrapper).toMatchSnapshot()
    })

})