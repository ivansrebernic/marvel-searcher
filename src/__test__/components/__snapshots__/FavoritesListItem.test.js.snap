// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FavoritesListItem> Renders correctly 1`] = `<styled.li />`;
